plugins/CoreAdminHome/lang/en.json

```diff
@@ -104,6 +104,10 @@
         "YouMayOptOutBis": "To make that choice, please click below to receive an opt-out cookie.",
         "OptingYouOut": "Opting you out, please wait...",
         "ProtocolNotDetectedCorrectly": "You are currently viewing Matomo over a secure SSL connection (using https), but Matomo could only detect a non secure connection on the server. ",
-        "ProtocolNotDetectedCorrectlySolution": "To make sure Matomo securely requests and serves your content over HTTPS, you may edit your %1$s file and either configure your proxy settings, or you may add the line %2$s below the %3$s section. %4$sLearn more%5$s"
+        "ProtocolNotDetectedCorrectlySolution": "To make sure Matomo securely requests and serves your content over HTTPS, you may edit your %1$s file and either configure your proxy settings, or you may add the line %2$s below the %3$s section. %4$sLearn more%5$s",
+        "MissingTrackingCodeEmailSubject": "No traffic for %s recorded, get started now",
+        "JsTrackingCodeMissingEmail1": "Just a quick note - we checked and your Matomo doesn't seem to have any recorded traffic for %s.",
```

Comment:
I just tested the feature and I think it may be good to start with a sentence like 

> A few days ago you created the site "$sitename" in [your Matomo Analytics]($piwikUrl).

Just as a reminder for them to understand what they did and why they receive that mail.